### DISCUSSION TITLE
The mysql password that comes with Milvus installation kit is wrong

### DISCUSSION THREAD

**Discussion by sayfeeling** [milvus-io/milvus#3843] [Q&A and General discussion]
从这里下载安装milvus的时候，初始化mysql，提示mysql有问题。
https://github.com/milvus-io/milvus-helm.git

提示如下：
Warning  Unhealthy  66s (x3 over 86s)    kubelet, adm-node2  Liveness probe failed: mysqladmin: [Warning] Using a password on the command line interface can be insecure.
mysqladmin: connect to server at 'localhost' failed
error: 'Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)'
Check that mysqld is running and that the socket: '/var/run/mysqld/mysqld.sock' exists!
  Normal   Killing    66s                 kubelet, adm-node2  Container my-release-mysql failed liveness probe, will be restarted
  Warning  Unhealthy  38s (x8 over 108s)  kubelet, adm-node2  Readiness probe failed: mysqladmin: [Warning] Using a password on the command line interface can be insecure.
mysqladmin: connect to server at 'localhost' failed
error: 'Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)'
Check that mysqld is running and that the socket: '/var/run/mysqld/mysqld.sock' exists!

请问是否是你们设置的密码有问题？

**Comment by kateshaowanjou** [comment]
Hi sayfeeling,
Please use English to communicate as GitHub is a platform open to a global audience. Your question might benefit others who don't speak Chinese, too. :)

**Comment by shiyu22** [comment]
Which version did you use? We don't meet this problem with Milvus-Helm 0.10.3.

And please make sure it doesn't conflict with your local MySQL service.
